### PR TITLE
Alert levels

### PR DESCRIPTION
**Pull request by OliverOtter**
## About the PR

Adds 5 new alert levels that admins can use. They can be used for RP/event reasons, and work similar to the Gamma/Epsilon alert, being unable to be changed by crew.
## Why / Balance

Can be used for rp purposes, and for the future. Ex: Code Lima can be auto called if cults where in game, and where making the nar'sie summoning rune.
## Technical details
NA

## Media

image
image
image
image
image

- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

N/A
**Changelog**

no cl no fun

### CODE REVIEW

**Comment by lzk228** [comment]
nuuuh

**Comment by UbaserB** [comment]
I feel like this is a bit too many presets. Maybe 1 or two sure, but this feels like bloat since admins can manually set up their own iirc

**Comment by mirrorcult** [comment]
these all seem extremely way too situational and memey to be actual prototypes in the game and not just uploaded

**Comment by UbaserB** [comment]
> these all seem extremely way too situational and memey to be actual prototypes in the game and not just uploaded

i would probably keep alert navy though, events with visitors are fairly common.

**Comment by lzk228** [comment]
> these all seem extremely way too situational and memey to be actual prototypes in the game and not just uploaded

they cant even be uploaded since name hardcoded #21525 :trollface:

**Comment by Clostyandr** [comment]
The last two is cringe. Other is OK, but call it normal i can't.

**Comment by IProduceWidgets** [comment]
> > these all seem extremely way too situational and memey to be actual prototypes in the game and not just uploaded
> 
> i would probably keep alert navy though, events with visitors are fairly common.

But why would a visitor require an alert level that changes the alarm colors and such? 

Many of these feel like a way to have recolored announcements, which would just be a nice feature in itself.

File: Resources/Locale/en-US/alert-levels/alert-levels.ftl [hunk 1]
```diff
@@ -34,3 +34,23 @@ alert-level-delta-instructions = Crewmembers are advised to listen to heads of s
...
+alert-level-brown = Brown
+alert-level-brown-announcement = The station is deemed to be very filthy. Grab cleaning tools and start cleaning around the station.
+alert-level-brown-instructions = Help clean the station.
+
+alert-level-cyan = Cyan
+alert-level-cyan-announcement = A unprobable time paradox casued by certain crewmembers has been detected in the timeline. Future Central Command is deploying a unit to fix the paradox.
+alert-level-cyan-instructions = Time paradox detected.
+
+alert-level-beige = Beige
+alert-level-beige-announcement = Central Command has seen the shit going on the station, and is just dissapointed in everyone on board. Everyone is required to feel shame.
```
Comment (Veritius) [inline Resources/Locale/en-US/alert-levels/alert-levels.ftl:51]: Spelling:
```suggestion
alert-level-beige-announcement = Central Command has seen the shit going on the station, and is just disappointed in everyone on board. Everyone is required to feel shame.
```

Funnier:
```suggestion
alert-level-beige-announcement = Central Command has seen the shit going on the station, and is just disappointed in everyone on board. Everyone is contractually obligated to feel shame.
```

File: Resources/Locale/en-US/alert-levels/alert-levels.ftl [hunk 2]
```diff
@@ -34,3 +34,23 @@ alert-level-delta-instructions = Crewmembers are advised to listen to heads of s
...
+alert-level-navy-announcement = Central Command has ordered the Navy security level on the station. Special guests will be visiting the station. Security is to be on guard and escort the guests as needed. Crewmembers should look professional and tidy their work places.
+alert-level-navy-instructions = Be on your best behaviour.
+
+alert-level-brown = Brown
+alert-level-brown-announcement = The station is deemed to be very filthy. Grab cleaning tools and start cleaning around the station.
+alert-level-brown-instructions = Help clean the station.
+
+alert-level-cyan = Cyan
+alert-level-cyan-announcement = A unprobable time paradox casued by certain crewmembers has been detected in the timeline. Future Central Command is deploying a unit to fix the paradox.
+alert-level-cyan-instructions = Time paradox detected.
```
Comment (Veritius) [inline Resources/Locale/en-US/alert-levels/alert-levels.ftl:48]: These aren't instructions.

File: Resources/Locale/en-US/alert-levels/alert-levels.ftl [hunk 3]
```diff
@@ -34,3 +34,23 @@ alert-level-delta-instructions = Crewmembers are advised to listen to heads of s
...
+
+alert-level-beige = Beige
+alert-level-beige-announcement = Central Command has seen the shit going on the station, and is just dissapointed in everyone on board. Everyone is required to feel shame.
+alert-level-beige-instructions = Feel shame, for you are a dissapointment to NT.
+
+alert-level-lima = Lima
+alert-level-lima-announcement = Central Command has ordered the Lima security level on the station. There is currently a supernatural or eldritch level abnormallity to the station. All staff on board is investigate and eliminate the abnormallity.
```
Comment (Veritius) [inline Resources/Locale/en-US/alert-levels/alert-levels.ftl:55]:
```suggestion
alert-level-lima-announcement = Central Command has ordered the Lima security level on the station. There is currently a supernatural or eldritch level abnormality to the station. All staff on board is investigate and eliminate the abnormality.
```

**Review by Veritius — CHANGES_REQUESTED** [review]

**Comment by metalgearsloth** [comment]
what mirror said nutria

**Pull request closed without merging**